CODEOWNERS

```diff
@@ -3,6 +3,7 @@
 # the last matching pattern has the most precedence.
 
 # Current maintainers
-
 * @jonathan-casarrubias @VMois.
 
+# Core team members from IBM
+* @bajtos @raymondfeng @kjdelisle @virkt25
```

Comment:
Not all of us will have bandwidth to server as code owners for this repository. Could you please remove these two lines for now?

Reply:
Yeah I can do that